horace_core/sqw/PixelData/@pixobj_combine_info/private/check_combo_arg_.m

```diff
@@ -0,0 +1,23 @@
+function obj = check_combo_arg_(obj)
+%CHECK_COMBO_ARG_ validate consistency of cellarray of pixels data and
+% cellarray of distributions, describing these pixels
+if isempty(obj.npix_list_) && ~isempty(obj.infiles_)
+    obj.do_check_combo_arg_ = false; % avoid calling this routine recursively
+    obj.npix_list = num2cell(obj.npix_each_file_);
```

Comment:
Why does a `check` routine change things?

Reply:
well -- may be it should be renamed. But it changes it in multiple places.